go.mod

```
@@ -0,0 +1,3 @@
+module rps
```

I forgot to say but traditionally your module would actually be the GitHub repo.


```suggestion
module github.com/Nicolhime/rock-paper-scissors
```